client/src/api/planters.js

```diff
@@ -6,14 +6,36 @@ import {
 import {session} from "../models/auth";
 
 export default {
-  getPlanter(id){
-    const query = `${process.env.REACT_APP_API_ROOT}/api/${getOrganization()}planter/${id}`;
-    return fetch(query, {
-      method: "GET",
-      headers: { 
-        "content-type": "application/json" ,
-        Authorization: session.token ,
-      },
+  getPlanter(id) {
+    const apiRoot = `${process.env.REACT_APP_API_ROOT}/api`;
+    const planterQuery = `${apiRoot}/${getOrganization()}planter/${id}`;
+    const registrationQuery = `${apiRoot}/planter_registration/?filter=where[planterId]=${id}`;
```

Comment:
same nit from the other place to use dashes in urls

Reply:
Fixed